src/sql/workbench/contrib/notebook/browser/markdownWYSIWYGLinkHander.ts

```diff
@@ -0,0 +1,91 @@
+import { URI } from 'vs/base/common/uri';
+import * as path from 'vs/base/common/path';
+
+export class MarkdownWYSIWYGLinkHandler {
```

Comment:
This class could use comments explaining what the different parts are supposed to be doing and why. For example `getLinkUrl` - what is this URL supposed to be for? What format is it going to be in? Is it going to be escaped or not? Stuff like that - just help make it so people know what to expect and don't have to read through the code for all this.

Reply:
Is there anything specifically about this that makes it a "WYSIWYG Link Handler` vs just a generic link helper? Maybe just something like `NotebookLinkHelper`?